modules/openapi-generator/src/main/resources/rust-server/models.mustache

```diff
@@ -296,6 +296,19 @@ pub struct {{{classname}}} {
             range(min = {{minimum}}),
             {{/maximum}}
         {{/minimum}}
+        {{#maxItems}}
+            {{#minItems}}
+            length(min = {{minItems}}, max = {{maxLength}}),
```

Comment:
maxLength should be replaced by maxItems.

caught this issue when testing with both minItems and maxItems

will fix it in https://github.com/OpenAPITools/openapi-generator/pull/17215

Reply:
Oops, that would be the result of having used `/Length/s//Items/` where `/Length/s//Items/g` was needed.